test/media-groups.test.js

```diff
@@ -726,6 +726,7 @@ function(assert) {
   assert.ok(this.mediaTypes[type].groups.sub2[1].playlistLoader,
     'playlistLoader created');
   assert.ok(this.mediaTypes[type].tracks.en, 'created text track');
+  assert.equal(this.mediaTypes[type].tracks.en.default, true, 'en track auto selected');
```

Comment:
This seems to be testing that the default property is set, rather than it being auto selected.

Reply:
I think that's fine? Actually selecting the caption by default is the job of Video.js, so, there's no need to test that behavior here (or override it). But we do need to make sure that the signal is passed to Video.js properly.